### PR TITLE
feat: draw oversimplified poker table

### PR DESCRIPTION
**Pull request by jackinf**
# Description
`/table` command outputs this:
image

### CODE REVIEW

File: pokerapp/model/pokerbotmodel.py [hunk 1]
```diff
@@ -172,7 +175,7 @@ def start(self, update: Update, context: CallbackContext) -> None:
                 chat_id=chat_id,
                 text=text,
             )
-            self._view.send_photo(chat_id=chat_id)
+            self._view.send_photo(chat_id=chat_id, photo=open("./assets/poker_hand.jpg", 'rb'))
```
Comment (kkoemets) [inline pokerapp/model/pokerbotmodel.py:178]: :beer:

File: pokerapp/utils/draw_poker_table.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+def __draw_circle(ax, center, radius, color='black', linewidth=1):
+    t = np.linspace(0, 2 * np.pi, 100)
+    x = center[0] + radius * np.cos(t)
+    y = center[1] + radius * np.sin(t)
+    ax.plot(x, y, color=color, linewidth=linewidth)
+
+
+def draw_poker_table(players: List[Player], current_player: Player):
+    assert 1 <= len(players) <= 6
```
Comment (kkoemets) [inline pokerapp/utils/draw_poker_table.py:32]: https://github.com/kkoemets/Poker-Telegram-Bot/blob/0c4988e22c9b6beae0b365dc182a204f52a49216/pokerapp/model/pokerbotmodel.py#L41

**Comment by kkoemets** [comment]
Thankss, great stuff for starters! Lots of stuff done, ground works for something like:
image

File: pokerapp/utils/draw_poker_table.py [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+
+def __draw_circle(ax, center, radius, color='black', linewidth=1):
+    t = np.linspace(0, 2 * np.pi, 100)
+    x = center[0] + radius * np.cos(t)
+    y = center[1] + radius * np.sin(t)
+    ax.plot(x, y, color=color, linewidth=linewidth)
+
+
+def draw_poker_table(players: List[Player], current_player: Player):
```
Comment (kkoemets) [inline pokerapp/utils/draw_poker_table.py:31]: prolly in the end Game entity needs to be passed to draw out everything the table, and pokerbotview.py is anyway coupled with Game entity

Comment (jackinf) [reply pokerapp/utils/draw_poker_table.py:31]: got it!

**Comment by jackinf** [comment]
image

**Comment by jackinf** [comment]
@kkoemets please check again, if you're happy with the result

**Comment by kkoemets** [comment]
Thanks, I will update the bot and let's see.

**Comment by kkoemets** [comment]
image
Looks nice!

**Pull request merged**